### ISSUE TITLE
Shishkina_Water - Cation fraction vs. Atomic proportion

### ISSUE THREAD

**Issue by PennyWieser** [kaylai/VESIcal#92]
Using the ShishkinaCarbon model, the calibration dataset for H2O-free experiments is well created

![image](https://user-images.githubusercontent.com/65668673/85200564-6a16dd80-b2f0-11ea-835b-0fd8e8e6d86a.png)

Using the ShishkinaWater model, carbon-free experiments do deviate a fair bit from the 1:1 line
![image](https://user-images.githubusercontent.com/65668673/85200569-7bf88080-b2f0-11ea-9904-fed25459a83d.png)
As a result, the mixed model match isn't great to a table in their paper for H2O-CO2-bearing experiments
![image](https://user-images.githubusercontent.com/65668673/85201599-a4847880-b2f8-11ea-9302-089ac1f3849d.png)

I've had a look at the equations in the code, and all looks well re no typos. However, in Shishkina, In their H2O equation, they refer to Na and K as "atomic proportions" while in the CO2 equation they use "cation fractions". And it doesn't really matter for the CO2 equation as obviously its  a fraction, while in H2O what you divide through by does matter. 

When I plot up their calibration dataset which is shown in their Fig. 7a, the only way that I can get their Na+K atomic proportion to match that graph is by using normalize_AdditionalVolatiles, and also removing CO2 and H2O from oxides, so it seems its by "atomic proportions" they mean "cation fractions ignoring volatiles"? 

I wasn't sure how to edit this in the code - so I pulled oxides and oxide mass into my Testing_Shishkina file so as not to screw up VESIcal to see if this worked. Would either of you be able to do this so I can test if this is the problem by running the H2O calibration dataset again using Na + K as norm w/o H2O and CO2 and see what PSats I get?

**Comment by simonwmatthews** [comment]
I think I've made the necessary change, but have not tested it beyond making sure the code still runs. Will push the update as soon as I've convinced gitkraken not to overwrite your jupyter notebooks..

**Comment by simonwmatthews** [comment]
@PennyWieser I've pushed it- let me know if it doesn't work and I'll fix it properly this evening.

**Comment by PennyWieser** [comment]
@simonwmatthews Thanks! Much better for H2O.
![image](https://user-images.githubusercontent.com/65668673/85207953-761c9280-b324-11ea-8680-160aa057071f.png)
And mixed CO2-H2O good considering noise in experiments at 5kbar (regression shown excluding those experiments)
![image](https://user-images.githubusercontent.com/65668673/85207964-8c2a5300-b324-11ea-8635-6700f1b3bca2.png)
and including them
![image](https://user-images.githubusercontent.com/65668673/85207971-9ea48c80-b324-11ea-9566-1dd64ce70188.png)

On a related note, do you have a preferred method of plotting a linear regression + a confidence interval using python (e.g., do you use LinearRegression() or something else? I've found a few on stackexchange but they are soo many lines of code vs. how easy it is in Matlab! Hoping there is a short version i've missed...

**Comment by kaylai** [comment]
This looks great! Nicely done, Penny! I've used [statsmodels](https://www.statsmodels.org/stable/regression.html) for doing multi-dimensional linear regressions, but that's useful for complicated cases (like major element modeling).

I would try scipy.optimize curve fit: https://stackoverflow.com/questions/22240280/least-squares-fit-to-a-straight-line-python-code

**Comment by simonwmatthews** [comment]
I often use scipy.optimize a lot, but there is also a linear regression tool in sklearn. There's also a package that does orthogonal-distance regression, though I forget what its called... I'm closing this as I think the problem is fixed?
